api/src/sockets/kots/DeploySocket.ts

```diff
@@ -131,8 +131,15 @@ export class KotsDeploySocketService {
                 kubectl_version: kotsAppSpec ? kotsAppSpec.kubectlVersion : "",
                 namespace: desiredNamespace,
                 manifests: b.toString("base64"),
+                previous_manifests: "",
               };
 
+              if (app.currentSequence) {
+                const previousRendered = await app.render(''+(app.currentSequence-1), `overlays/downstreams/${cluster.title}`);
```

Comment:
Will the previous sequence always be the previous version deployed? What happens if someone makes multiple edits, saving them each time, and then deploys? Will it only remove resources dropped by the last settings change?